site/_docs/reference.md

```diff
@@ -2852,6 +2855,7 @@ BigQuery's type system uses confusingly different names for types and functions:
 | b o p | TRANSLATE(expr, fromString, toString)      | Returns *expr* with all occurrences of each character in *fromString* replaced by its corresponding character in *toString*. Characters in *expr* that are not in *fromString* are not replaced
 | b | TRUNC(numeric1 [, numeric2 ])                  | Truncates *numeric1* to optionally *numeric2* (if not specified 0) places right to the decimal point
 | q | TRY_CAST(value AS type)                        | Converts *value* to *type*, returning NULL if conversion fails
+| h s | UNHEX(varchar)                               | Converts a hexadecimal-encoded *varchar* into bytes
```

Comment:
what happens if this fail?

Reply:
Good catch, in Hive and Spark, it returns `NULL`. I will add test cases for invalid inputs.

Reply:
Can you make doc more consistent with other functions; they don't have parameters called `*varchar*`

Doc should explicitly say what happens on failure